client/components/Pedalboards/Pedalboard/Pedalboard.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Button, Card, Image } from 'semantic-ui-react'
+
+const pedalBoard = ({ price, brand, model, description, image }) => {
+  return (
+    <Card>
+      <Card.Content>
+        <Image src={image} />
```

Comment:
There is something off with your cards. If you look closely, you can see that the product name on each card is at a slightly different vertical position because the images have different heights. You need to make them all uniform.